crates/muvm/src/guest/x11.rs

```diff
@@ -9,14 +9,14 @@ use byteorder::{BigEndian, ReadBytesExt, WriteBytesExt};
 
 use crate::guest::x11bridge::start_x11bridge;
 
-pub fn setup_x11_forwarding<P>(run_path: P) -> Result<bool>
+pub fn setup_x11_forwarding<P>(run_path: P, host_display: Option<&str>) -> Result<bool>
 where
     P: AsRef<Path>,
 {
     // Set by muvm if DISPLAY was provided from the host.
-    let host_display = match env::var("HOST_DISPLAY") {
-        Ok(d) => d,
-        Err(_) => return Ok(false),
+    let host_display = match host_display {
```

Comment:
Per a previous comment, this is now required so it should not longer be an `Option<T>` at all.